### PR TITLE
fix nebula--cpp example cmake

### PR DESCRIPTION
**Pull request by Winging**
Fix cmake for disable cxx11 abi in example

### CODE REVIEW

File: examples/CMakeLists.txt [hunk 1]
```diff
@@ -16,7 +16,7 @@ endif()
 option(DISABLE_CXX11_ABI                "Whether to disable cxx11 abi" OFF)
 
 # Is abi 11
-if (${DISABLE_CXX11_ABI})
+if (${DISABLE_CXX11_ABI} STREQUAL "ON")
```
Comment (Shylock-Hg) [inline examples/CMakeLists.txt]: `if(DISABLE_CXX11_ABI)` is better.

**Comment by Shylock-Hg** [comment]
Plz check cla

File: examples/CMakeLists.txt (hunk 1 above)
Comment (Winging) [reply examples/CMakeLists.txt]: Sure, fixed.

**Review by Shylock-Hg — APPROVED** [review]

**Comment by jamieliu1023** [comment]
@Winging  Thanks for your contribution to the Nebula Graph community! This is Jamie with Nebula and I'd like to email you the Nebula Contributor certificate and ship you a mug to mark this special moment. Could you please kindly reach me via jamie.liu(at)vesoft.com? Again, thanks for being a part of the Nebula community!

**Pull request merged**